<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно работника</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(120,120,120);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_quit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,255,255);
color: rgb(0,0,0);
font: 18pt \&quot;Arial\&quot;;</string>
    </property>
    <property name="text">
     <string>ВЫХОД</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_work">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>291</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,255,255);
color: rgb(0,0,0);
font: 18pt \&quot;Arial\&quot;;</string>
    </property>
    <property name="text">
     <string>НАЖАТЬ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(120,120,120);
color: rgb(0,0,0);
font: 18pt \&quot;Arial\&quot;;</string>
    </property>
    <property name="text">
     <string>Ваш результат:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_greetings">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(120,120,120);
color: rgb(0,0,0);
font: 18pt \&quot;Arial\&quot;;</string>
    </property>
    <property name="text">
     <string>Добро пожаловать</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
